### PR TITLE
Upgrading the Gradle version to 6.9

### PR DESCRIPTION
**Pull request by suztomo**
The current Gradle version 4.9 is too old. Let's upgrade it.

- Upgrading the Gradle wrapper version to 6.9 via `./gradlew wrapper --gradle-version 6.9`.
- Removed the built-in 'wrapper' task.
- Upgraded GoogleJavaFormat plugin version (because 0.2 didn't work with Gradle 6)
- Applied GoogleJavaFormat plugin to some of the Java code.

Version 6.9 is the latest version in 6.X releases. https://gradle.org/releases/ . Not using 7.X because https://github.com/GoogleCloudPlatform/endpoints-framework-gradle-plugin/issues/107 says some API has been removed in 7.0.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,8 +46,8 @@ test {
   }
 }
 
-task wrapper(type: Wrapper) {
-  gradleVersion = '4.0'
+task {
+  gradleVersion = '6.9'
```
Comment (suztomo) [inline build.gradle]: As per https://docs.gradle.org/4.8/release-notes.html#overwriting-gradle%E2%80%99s-built-in-tasks.

Do we need this?

File: build.gradle [hunk 2]
```diff
@@ -46,10 +46,6 @@ test {
   }
 }
 
-task wrapper(type: Wrapper) {
-  gradleVersion = '4.0'
-}
-
```
Comment (suztomo) [inline build.gradle:52]: The build-in task cannot be overridden any more. https://docs.gradle.org/5.2.1/userguide/upgrading_version_4.html#deprecations_4.8 This PR needs to fix this.

The built-in wrapper task allows us to configure "wrapper" task's behavior. https://docs.gradle.org/6.9/userguide/gradle_wrapper.html#customizing_wrapper
Specifying the Gradle version here for "./gradlew wrapper" while `gradle/wrapper/gradle-wrapper.properties` gives a version does not make sense. So I'm removing these lines.

**Comment by suztomo** [comment]
```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':verifyGoogleJavaFormat'.
...
Caused by: java.lang.NoSuchFieldError: source
	at com.github.sherter.googlejavaformatgradleplugin.FormatTask.hasSources(FormatTask.java:12)
	at com.github.sherter.googlejavaformatgradleplugin.TaskConfigurator.configureFormatTask(TaskConfigurator.java:22)
```

File: src/test/resources/projects/clientserver/server/src/main/java/com/example/Test.java [hunk 3]
```diff
@@ -22,11 +22,10 @@
...
-  name = "testApi",
-  version = "v1",
-  namespace =
-      @ApiNamespace(ownerDomain = "example.com", ownerName = "example.com", packagePath = "")
-)
+    name = "testApi",
+    version = "v1",
+    namespace =
+        @ApiNamespace(ownerDomain = "example.com", ownerName = "example.com", packagePath = ""))
 public class Test {
```
Comment (suztomo) [inline src/test/resources/projects/clientserver/server/src/main/java/com/example/Test.java:29]: `./gradlew :googleJavaFormat ` updated this file.

File: src/test/resources/projects/clientserver/server/src/main/java/com/example/Test2.java [hunk 4]
```diff
@@ -22,11 +22,10 @@
...
 @Api(
-  name = "testApi2",
-  version = "v1",
-  namespace =
-      @ApiNamespace(ownerDomain = "example.com", ownerName = "example.com", packagePath = "")
-)
+    name = "testApi2",
+    version = "v1",
+    namespace =
+        @ApiNamespace(ownerDomain = "example.com", ownerName = "example.com", packagePath = ""))
```
Comment (suztomo) [inline src/test/resources/projects/clientserver/server/src/main/java/com/example/Test2.java:28]: `./gradlew :googleJavaFormat ` updated this file.

File: src/test/resources/projects/server/src/main/java/com/example/Test.java [hunk 5]
```diff
@@ -22,11 +22,10 @@
...
 @Api(
-  name = "testApi",
-  version = "v1",
-  namespace =
-      @ApiNamespace(ownerDomain = "example.com", ownerName = "example.com", packagePath = "")
-)
+    name = "testApi",
+    version = "v1",
+    namespace =
+        @ApiNamespace(ownerDomain = "example.com", ownerName = "example.com", packagePath = ""))
```
Comment (suztomo) [inline src/test/resources/projects/server/src/main/java/com/example/Test.java:28]: `./gradlew :googleJavaFormat ` updated this file.

File: build.gradle [hunk 6]
```diff
@@ -3,7 +3,7 @@ plugins {
   id 'groovy'
   id 'maven'
   id 'java-gradle-plugin'
-  id 'com.github.sherter.google-java-format' version '0.6'
+  id 'com.github.sherter.google-java-format' version '0.8'
```
Comment (suztomo) [inline build.gradle:6]: GoogleJavaFormat plugin 0.9 (latest) complains minimum Java version:

> com/google/googlejavaformat/java/JavaFormatterOptions$Style has
been compiled by a more recent version of the Java Runtime (class
file version 55.0), this version of the Java Runtime only recognizes
class file versions up to 52.0

Therefore choosing the second latest version 0.8 that does not
complain about the Java version.

**Review by elharo — APPROVED** [review]

**Pull request merged**